app/assets/javascripts/think_feel_do_dashboard/participants/_form.js

```diff
@@ -0,0 +1,150 @@
+(function() {
+  'use strict';
+
+  var ParticipantForm = {};
+  var PASSPHRASE_LENGTH = 4;
+
+  function getRandomInt(min, max) {
+    return Math.floor(Math.random() * (max - min)) + min;
+  }
+
+  ParticipantForm.generatePassword = function() {
+    var password = [];
+
+    for (var i = 0; i < PASSPHRASE_LENGTH; i++) {
+      password.push(words()[getRandomInt(0, words().length)]);
+    }
+
+    this.password = password.join(' ');
```

Comment:
Does ```this``` in this case, refer to the global context, or is the value set somewhere outside of this function? I noticed you are using strict mode, so I'm assuming ```this``` is being set somewhere else.